bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -480,9 +517,25 @@ public String toString() {
      */
     public void asyncReadEntries(long firstEntry, long lastEntry,
                                  ReadCallback cb, Object ctx) {
+
         // Little sanity check
-        if (firstEntry < 0 || lastEntry > lastAddConfirmed
-                || firstEntry > lastEntry) {
+        if (firstEntry < 0 || firstEntry > lastEntry) {
+            cb.readComplete(BKException.Code.ReadException, this, null, ctx);
```

Comment:
reverted back the changes made to asyncReadEntries. The client application should call asyncReadExplicitLastConfirmed/readExplicitLastConfirmed to update the lac of the ledgerHandle.